src/test/bench/rt-messaging-ping-pong.rs

```diff
@@ -63,17 +63,15 @@ fn ping_pong_bench(n: uint, m: uint) {
 
 
 fn main() {
-
-    let args = os::args();
-    let args = args;
+    let mut args = env::args();
     let n = if args.len() == 3 {
-        args[1].parse::<uint>().unwrap()
+        args.nth(1).unwrap().parse::<usize>().unwrap()
     } else {
         10000
     };
 
     let m = if args.len() == 3 {
-        args[2].parse::<uint>().unwrap()
+        args.next().unwrap().parse::<usize>().unwrap()
```

Comment:
This one is not as obvious as it was before, which possibly makes refactoring this file a little bit harder if we ever introduce breaking changes to `env::args` in the future.

Not sure how important it is, though.